vagrantfile-windows.template

```diff
@@ -4,6 +4,7 @@
 Vagrant.configure(2) do |config|
   config.vm.guest = :windows
   config.vm.communicator = "winrm"
+  config.vm.network :forwarded_port, guest: 3389, host: 3389, id: 'rdp', auto_correct: true
```

Comment:
Nah, you don't need that.  the `auto_correct: true` handles it for us per the documentation: https://www.vagrantup.com/docs/networking/forwarded_ports.html